subsys/bluetooth/mesh/cfg_srv.c

```diff
@@ -2442,7 +2442,7 @@ static void mod_reset(struct bt_mesh_model *mod, struct bt_mesh_elem *elem,
 	}
 }
 
-void bt_mesh_cfg_reset(void)
+void bt_mesh_model_reset(void)
 {
 	bt_mesh_model_foreach(mod_reset, NULL);
 }
```

Comment:
With this name change, it probably makes sense to move this function and its model callback to access as well, where it really belongs

Reply:
Yeah, I thought the same initially. But it has a dependency with some code in config server. I'll need to convert one function to internal-public in config server.

Reply:
Right, `mod_sub_list_clear`. Then, don't bother :) I've had a task in my backlog for moving all those access config functions into their own local API for a long time. They should eventually have an API like cfg.h, but for access, so we can get rid of this config model self-configuration business once and for all.

Better to deal with these all in one batch than pick them off one by one.